STEP/AssVisitor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using STEP.AST;
+using STEP.AST.Nodes;
+
+namespace STEP;
+
+public class AssVisitor : TypeVisitor
+{
+    public AssVisitor(ISymbolTable symbolTable)
+    {
+        _symbolTable = symbolTable;
+    }
```

Comment:
Man kan også gøre SymbolTable til en protected property på TypeVisitor, så den automatisk er delt med alle subclasses

Reply:
kræver det ikke at symbol table bliver initialized på type visitor så, i stedet for injected gennem constructor?